### PR TITLE
Add capabilities file

### PR DESCRIPTION
**Pull request by yuga-cb**
### What changed? Why?
Add capabilities file to track which verb x network is supported per wallet type.

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: CAPABILITIES.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| Trade         |      ❌      |      ✅      |        ❌        |        ❌        |
+| Faucet        |      ✅      |      ❌      |        ✅        |        ❌        |
+| Server-Signer |      ✅      |      ✅      |        ❌        |        ❌        |
+
+## End-User Wallets
+
+| Concept    | Base-Sepolia | Base-Mainnet | Ethereum-Holesky | Ethereum-Mainnet |
+| ---------- | :----------: | :----------: | :--------------: | :--------------: |
+| Stake [^1] |      ❌      |      ❌      |        ✅        |        ✅        |
+| Stake [^1] |      ❌      |      ❌      |        ✅        |        ✅        |
```
Comment (alex-stone) [inline CAPABILITIES.md:21]: Is there an intended difference between these two rows?

File: CAPABILITIES.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| Send          |      ✅      |      ✅      |        ❌        |        ❌        |
+| Trade         |      ❌      |      ✅      |        ❌        |        ❌        |
+| Faucet        |      ✅      |      ❌      |        ✅        |        ❌        |
+| Server-Signer |      ✅      |      ✅      |        ❌        |        ❌        |
+
+## End-User Wallets
+
+| Concept            | Base-Sepolia | Base-Mainnet | Ethereum-Holesky | Ethereum-Mainnet |
+| ------------------ | :----------: | :----------: | :--------------: | :--------------: |
+| External Addresses |      ❌      |      ❌      |        ✅        |        ✅        |
```
Comment (alex-stone) [inline CAPABILITIES.md:20]: External Address are supported on `base-sepolia` and `base-mainnet` as well for balances purposes

Comment (jazz-cb) [reply CAPABILITIES.md:20]: Should call this "Balances" instead of Addresses in this table and above?

**Review by jazz-cb — APPROVED** [review]

File: CAPABILITIES.md (hunk 2 above)
Comment (yuga-cb) [reply CAPABILITIES.md]: Thanks for catching. Fixed.

**Pull request merged**